src/components/views/causes/create/CauseReviewStep.tsx

```
@@ -0,0 +1,901 @@
+							{formatMessage({
+								id: 'label.cause.title',
+							})}
+						</TitleLable>
+						<TitleValue>{title}</TitleValue>
+						<TitleLable>
+							{formatMessage({
+								id: 'label.cause.create_description',
+							})}
+						</TitleLable>
+						<DescriptionValue
+							dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)